logrotation/services/confstore.yaml

```diff
@@ -0,0 +1,4 @@
+cortx:
```

Comment:
This file can move to startup itself as its related to that and going to be consumed there

Reply:
yes moved it. 👍